exercises/exercise_5/README.md

```diff
@@ -76,20 +76,27 @@ To update the vote and results services, let's do the following:
 
 3. Do the same thing for the `results` service, but use `results.enabled` as the parameter name.
 
-4. Use the `docker app inspect` command and set the `vote.enabled` parameter to false. You'll see that the `vote` service is not included, as well as the `vote.enabled` parameter is set to false.
+4. Do not forget to build the application image.
+
+```sh
+$ docker app build voting-app -t <your-hub-username>/voting-app.dockerapp
+```
+
+5. Use the `docker app inspect` command. You'll see the `vote.enabled` and `results.enabled` parameters are set to the default value `true`.
 
     <details>
       <summary>Sample Output</summary>
 
     ```console
-    $ docker app inspect -s vote.enabled=false
+    $ docker app inspect <your-hub-username>/voting-app.dockerapp
```

Comment:
This is the output of `docker app inspect --pretty` (kinda)

Reply:
Fixed